models/identity-stitcher.js

```diff
@@ -235,10 +235,10 @@ const isSameSame = (user, other) => {
   // If sso_guid exists on both, and is different, it's not the same
   if (user.has_sso_guid && other.has_sso_guid && matches['sso_guid'].length === 0) return false
 
-  // If device_idfa match, then it's the same user
-  if (matches['device_idfa'].length > 0) return true
+  // If device_idfa or mcid match, then it's the same user
+  if (matches['device_idfa'].length > 0 || matches['mcid'].length > 0) return true
 
-  // If device_idfa exists on both and is different, t's not the same
+  // If device_idfa exists on both and is different, it's not the same
   if (user.has_device_idfa && other.has_device_idfa && matches['device_idfa'].length === 0) return false
```

Comment:
Is there any desire to do this with mcid?

Reply:
I chose not to, to allow it to be used in the next "if 2 match" block. With the ability to clear cookies, I figured it's more likely to exist in both and not match, even if other data still does. It's less likely `device_idfa` will change. I could more likely see removing the `device_idfa` check.